### PR TITLE
Abstracts for chapters 1, 3, 11, and 13

### PR DESCRIPTION
**Pull request by robertodr**
## Status

- [x]  Ready to go

### CODE REVIEW

File: chapter-01/recipe-04/README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Controlling compilation with conditionals
 
-Abstract to be written ...
+This recipe demonstrates how to use the `if-else-endif` conditional construct in `CMakeLists.txt`.
```
Comment (TheErk) [inline chapter-01/recipe-04/README.md]: Could be nice to link to the doc when appropriate.
https://cmake.org/cmake/help/latest/command/if.html

don't you think?

File: chapter-01/recipe-09/README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Setting the standard for the language
 
-Abstract to be written ...
+We will show various mechanism to require and set a specific compiler standard
```
Comment (TheErk) [inline chapter-01/recipe-09/README.md]: I think the title is better. I would have said something like:

We will show various mechanism to require and set a specific language standard in a compiler-agnostic way for our project.

**Comment by TheErk** [comment]
I didn't see this one was already merged sorry about that.
What do you think of adding cmake doc link each time its possible? Would you like me to try it or do you think it's useless. I don't want to slow you down to the finish line.

Can you explain me why you have to have both README.md and abstract.md ?

**Comment by robertodr** [comment]
All `README.md` files in the repo (at all levels) are autogeneratred by `contributing/generating-readmes.py` script by collating `title.txt` and `abstract.md` In this way, moving around entire folders can be done without having to adjust cross-references by hand. Long story short: you just need to review `title.txt` and `abstract.md`.

I was being lazy with my abstracts. I agree about pointing to the docs, great suggestion! I will do that in my next batch of abstracts and correct the ones that have been merged already.

**Comment by bast** [comment]
Dear Eric, I think links are a very good suggestion. Every PR is welcome and I will also consider this when adding more chapters tonight. We are expecting to mint the `v1.0` any day now but probably still have couple of days and until then we want to get the abstracts in.

About the `README.md` and `abstract.md`: only the latter is hand-edited, `README.md` is generated:
https://github.com/dev-cafe/cmake-cookbook/tree/master/contributing#updating-the-table-of-contents-and-generate-readme-files

The reason is that then we assemble not only the abstract, but also titles, and links.

**Comment by TheErk** [comment]
Ok thank you for the explanation. I'll try a PR on the already merged abstracts tonight.
That way I won't step on your foot while you progress on other chapters/recipes

**Comment by bast** [comment]
Thank you! The PRs are super welcome and this is very kind.

**Pull request merged**